usagov/usagov-2021
# USAGOV-2094: Fixing a high contrast issue with SVG

In Mobile view with High Contrast mode the SM icons are missing


https://cm-jira.usa.gov/browse/USAGOV-2094

## Description
The social media icon (SVG file) had a png/bitmap embedded inside of it - this code change removes the embedded PNG and instead replaces it with a true SVG vectorgram. This fixes a problem with the high-contrast renderer for the native browser on Android phones.

## Type of Changes


## Testing Instructions
This must be tested in the *native* browser (not Chrome) on an Adroid phone. After this merge, we simply 
* load the front page
* scroll to the bottom
* confirm the Twitter/X icon is visible in this normal-mode
* turn on high-contrast mode with: Accessibility -> Display Size and Color -> High Contrast Text
* confirm the Twitter/X social media icon is still rendering as expected.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
